tests/s390x_tests/TOOL_s390_chchp/consoletest_TOOL_s390_chchp.pm

```diff
@@ -0,0 +1,38 @@
+# SUSE’s openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC                                                                                    
+# Copyright (C) 2018 IBM Corp.
```

Comment:
It's ok to just use the IBM copyright from 2018. I do not see any SUSE content left ;)